atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractKeyValueService.java

```diff
@@ -76,6 +80,7 @@
             + " while doing a write to {}. Attempting to batch anyways.";
 
     protected ExecutorService executor;
+    protected volatile KeyValueServiceArgSupplier argSupplier = KeyValueServiceArgSupplier.ALL_UNSAFE;
```

Comment:
nit: maybe rename class/variable to `KeyValueServiceLoggingArgSupplier loggingArgs`. then the log lines look like `loggingArgs.tableRef(...)`